<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="main_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="title" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="decYearButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/popular/icons8-back-64.png</normaloff>:/icons/popular/icons8-back-64.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="date_info">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="_">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="incYearButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/popular/icons8-forward-64.png</normaloff>:/icons/popular/icons8-forward-64.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="button_container" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="m1">
           <property name="text">
            <string>OCAK</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="m2">
           <property name="text">
            <string>ŞUBAT</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="m3">
           <property name="text">
            <string>MART</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="m4">
           <property name="text">
            <string>NİSAN</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="m5">
           <property name="text">
            <string>MAYIS</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="m6">
           <property name="text">
            <string>HAZİRAN</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="m7">
           <property name="text">
            <string>TEMMUZ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="m8">
           <property name="text">
            <string>AĞUSTOS</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="m9">
           <property name="text">
            <string>EYLÜL</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="m10">
           <property name="text">
            <string>EKİM</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="m11">
           <property name="text">
            <string>KASIM</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="m12">
           <property name="text">
            <string>ARALIK</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="retTodayButton">
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/sut/arrow/dark/return-icon-9.png</normaloff>:/sut/arrow/dark/return-icon-9.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/convertor.qrc"/>
 </resources>
 <connections/>
</ui>
